### ISSUE TITLE
problem with gradle:1.3.0-beta1

### ISSUE THREAD

**Issue by AndroidGecko** [moallemi/gradle-advanced-build-version#11]
Thanks for the cool lib
Just wanted to flag an issue where you use ```renameOutput true``` and new gradle plugin 1.3.0-beta1

after adding
classpath 'com.android.tools.build:gradle:1.3.0-beta1'

the zipAlign task cannot find the input file because its renamed

```
FAILURE: Build failed with an exception.
A problem was found with the configuration of task ':app:zipalignDebug'.
File 'app/build/outputs/apk/app-debug-unaligned.apk' specified for property 'inputFile' does not exist.
```

**Comment by chelexey** [comment]
I have same problem with gradle:1.3.0 (final)

**Comment by Ewg777** [comment]
+1

**Comment by moallemi** [comment]
It seems that this is a bug in gradle 1.3.0 and has been fixed in gradle 1.3.1.
https://code.google.com/p/android/issues/detail?id=182248

For now you can use gradle 1.3.1 until I fix this issue
